### DISCUSSION TITLE
Unable to find the default Grid-Stat config file

### DISCUSSION THREAD

**Discussion by jwraby** [dtcenter/METplus#2132] [Training] [answered]
In the EC2 instance for the METplus training conducted earlier today, I tried to copy the default config file and rename it per the instructions at: https: //dtcenter.org/metplus-practical-session-guide-version-5-0/session-1-metplus-setupgrid-grid/met-tool-grid-stat/configure. The command I used was: cp ${MET_BUILD_BASE}/share/met/config/GridStatConfig_default GridStatConfig_tutorial. 
echo ${MET_BUILD_BASE} contains the path /opt. I don't see the /share directory in /opt. Can you provide the location of the default config file? Thanks.

**Comment by hankenstein2** [comment]
Hi again, this one is on me, there is/was an error in the EC2 setup which I will fix shortly (done). If you really want to know where it came from it was the directory that I build MET in which is /opt/METplus/build_components.

So /opt/share is now actually the correct location.

**Reply by jwraby** [reply, marked as answer]
Hi Hank - I'm logged into the cloud ec2-user account and navigated to /opt, but I still don't see the /share directory. I see directories like 11.0.0, 4.1, METcalcpy, METplus, bin, met, tomcat. I did find the config file at /opt/METplus/build_components/share/met/config per your reply above.

**Reply by hankenstein2** [reply]
Hi John, are you set with the config file you found?

**Reply by jwraby** [reply]
Hi Hank - Thanks for following up. Yes, I found it and also successfully copied/renamed/edited it using a different command line editor (emacs) which enables copy/paste from external text.

**Reply by hankenstein2** [reply]
Great, I will mark this as closed, thank you!
